src/amber/environment/logging.cr

```diff
@@ -57,7 +57,7 @@ module Amber::Environment
       @context = logging["context"].as(Array(String))
     end
 
-    def severity : Logger::Severity
+    def severity : Log::Severity
       SEVERITY_MAP[@severity]
```

Comment:
This can be replace with `Logger::Severity.parse @severity` allowing to remove the map